usersync/usersyncers/ttx.go

```diff
@@ -0,0 +1,17 @@
+package usersyncers
+
+import (
+	"net/url"
+	"strings"
+)
+
+func NewTtxSyncer(externalURL string, userSyncUrl string, partnerId string) *syncer {
```

Comment:
To rephrase it: is there a way for this function to indicate that no sync is actually possible?  (Ex: return `null`, or something like that?)

Basically, we want to avoid having the browser make a request if we already know the request will fail.

Reply:
I can add in a check if the PartnerId is missing, then I just return an empty redirect url

Reply:
k -- we should follow up w/ Prebid to see if they have any recommendations around this.